server/src/plugin/runtime/python-worker.ts

```diff
@@ -111,6 +111,14 @@ export class PythonRuntimeWorker extends ChildProcessWorker {
 
             this.worker.stdout.pipe(this.stdout);
             this.worker.stderr.pipe(this.stderr);
```

Comment:
I think the pipe call automatically sends this.worker.stderr errors to this.stderr.

Reply:
I believe either 3 or 4, the socket that raised the error had this specific event handler configured: https://github.com/nodejs/node/blob/9f9069d3130dc336a86271ca4e3c36e9b87cce90/lib/internal/child_process.js#L456

Reply:
gotcha, that makes sense, as 1 and 2 (stdout/stderr) are handled via pipe.